src/main/java/com/example/jpashop/service/CartService.java

```diff
@@ -19,32 +19,38 @@ public class CartService {
     private CartRepository cartRepository;
 
     public CartItemDto addItemToCart(CartItemDto cartItemDto) {
+        log.info("dfa");
         CartItem cartItem = new CartItem(
                 cartItemDto.getProductName(),
                 cartItemDto.getQuantity(),
                 cartItemDto.getPrice()
         );
-
+        log.info("df");
         CartItem savedItem = cartRepository.save(cartItem);
         return toDto(savedItem);
     }
 
     public List<CartItemDto> getCartItems() {
+        log.info("df");
         List<CartItem> cartItems = cartRepository.findAll();
+        log.info("df");
         return cartItems.stream().map(this::toDto).collect(Collectors.toList());
     }
```

Comment:
아래 같이 작성하는 게 더 가독성이 좋을 것 같아요.
```suggestion
public List<CartItemDto> getCartItems() {
    return cartRepository.findAll().stream()
                    .map(this::toDto)
                    .toList();
}
```